### PR TITLE
Admin Navbar link

### PR DESCRIPTION
**Pull request by agriffard**
Related to #14435

Put the admin link to visit the front site in its own .cshtml and render the shape in the Layout.

### CODE REVIEW

**Comment by agriffard** [comment]
This way, I can easily create an `Admin template` if I want to change it (use `target="_blank"`):

image

File: src/OrchardCore.Themes/TheAdmin/Views/NavbarLink.cshtml [hunk 1]
```diff
@@ -0,0 +1,5 @@
+<li class="nav-item">
+    <a href="@Url.Content("~/")" class="nav-link" data-bs-toggle="tooltip" data-bs-placement="bottom" title="@T["Visit Site"]" role="button">
+        <i class="fa-solid fa-fw fa-external-link" aria-hidden="true"></i>
+    </a>
+</li>
```
Comment (MikeAlhayek) [inline src/OrchardCore.Themes/TheAdmin/Views/NavbarLink.cshtml:5]: new line

**Review by MikeAlhayek — APPROVED** [review]
Good solution. But, we should consider changing the menu entirely using a shape like we did in UserMenu.

**Review by hishamco — CHANGES_REQUESTED** [review]
Just add a new line at the end

File: src/OrchardCore.Themes/TheAdmin/Views/NavbarLink.cshtml (hunk 1 above)
Comment (hishamco) [inline src/OrchardCore.Themes/TheAdmin/Views/NavbarLink.cshtml:5]:
```suggestion
</li>

```

**Review by hishamco — CHANGES_REQUESTED** [review]

File: src/OrchardCore.Themes/TheAdmin/Views/NavbarLink.cshtml (hunk 1 above)
Comment (agriffard) [reply src/OrchardCore.Themes/TheAdmin/Views/NavbarLink.cshtml:5]: Done in 7ef8072332bae0144fde567e9e229eac1b3ac793

**Review by hishamco — APPROVED** [review]

**Pull request merged**